doc/release/1.9.0-notes.rst

```diff
@@ -70,60 +145,806 @@ New features
 
 `scipy.stats` improvements
 ==========================
-
-Hypothesis Tests
-----------------
-
-
-Sample statistics
------------------
-
-
-Statistical Distributions
--------------------------
-
+- `scipy.stats.monte_carlo_test` performs one-sample Monte Carlo hypothesis
+tests to assess whether a sample was drawn from a given distribution. Besides
+reproducing the results of hypothesis tests like `scipy.stats.ks_1samp`,
+`scipy.stats.normaltest`, and `scipy.stats.cramervonmises` without small sample
+size limitations, it makes it possible to perform similar tests using arbitrary
+statistics and distributions.
+
+- Several ``scipy.stats`` functions support new ``axis`` (integer or tuple of
+integers) and ``nan_policy`` ('raise', 'omit', or 'propagate') arguments.
+These functions also support masked arrays as inputs, even if they do not have
+a ``scipy.stats.mstats`` counterpart. Edge cases for multidimensional arrays,
+such as when axis-slices have no unmasked elements or entire inputs are of
+size zero, are handled consistently. Documentation for all these standard
+features is available here (to be linked).
+
+- Add a ``weight`` parameter to `scipy.stats.hmean`.
+
+- Several improvements have been made to `scipy.stats.levy_stable`. Substantial
+improvement has been made for numerical evaluation of the pdf and cdf,
+resolving [#12658](https://github.com/scipy/scipy/issues/12658) and
+[#14944](https://github.com/scipy/scipy/issues/14994). The improvement is
+particularly dramatic for stability parameter ``alpha`` close to or equal to 1
+and for ``alpha`` below but approaching its maximum value of 2. The alternative
+fast Fourier transform based method for pdf calculation has also been updated
+to use the approach of Wang and Zhang from their 2008 conference paper
+*Simpson’s rule based FFT method to compute densities of stable distribution*,
+making this method more competitive with the default method. In addition,
+users now have the option to change the parametrization of the Levy Stable
+distribution to Nolan's "S0" parametrization which is used internally by
+SciPy's pdf and cdf implementations. The "S0"  parametrization is described in
+Nolan's paper [*Numerical calculation of stable densities and distribution
+functions*](https://doi.org/10.1080/15326349708807450) upon which SciPy's
+implementation is based. "S0" has the advantage that ``delta`` and ``gamma``
+are proper location and scale parameters. With ``delta`` and ``gamma`` fixed,
+the location and scale of the resulting distribution remain unchanged as
+``alpha`` and ``beta`` change. This is not the case for the default "S1"
+parametrization. Finally, more options have been exposed to allow users to
+trade off between runtime and accuracy for both the default and FFT methods of
+pdf and cdf calculation. More information can be found in the documentation
+here (to be linked).
+
+- Added `scipy.stats.fit` for fitting discrete and continuous distributions to
```

Comment:
Just noticing this now, `fit` is a _very_ generic name. I can't remember it being proposed/discussed. Will comment on gh-15436.